### PR TITLE
 fix(Avatar): remove array of images returning

### PR DESCRIPTION
**Pull request by bfbiggs**

### CODE REVIEW

File: src/lib/Avatar/index.js [hunk 1]
```diff
@@ -84,26 +84,18 @@ export default class Avatar extends React.Component {
...
       if (src && !isImageErrored) {
-        const imgChildren = [];
         // image is not loaded and title is provided
-        if (title && !isImageLoaded) {
-          imgChildren.push(getLetter());
-        }
-        imgChildren.push(
-          <img
+        return (title && isImageLoaded)
+          ? <img
```
Comment (shashankcs090) [inline src/lib/Avatar/index.js]: The image would not render at all, if src prop is present.

Comment (bfbiggs) [reply src/lib/Avatar/index.js]: if src is present and it's loaded - img will be present, if not the title will show.

File: src/lib/Avatar/index.js [hunk 2]
```diff
@@ -73,7 +73,10 @@ export default class Avatar extends React.Component {
       return (
         <span
           key='letter'
-          className='cui-avatar__letter'
+          className={
+            'cui-avatar__letter' +
+            `${(isImageLoaded && ` cui-avatar__img--hidden`) || ''}`
```
Comment (shashankcs090) [inline src/lib/Avatar/index.js]: @bfbiggs looks like the change u did in core is enough to fix the issue.
The issue was, there was class .cui-avatar--hidden{ display: none;} in core but the class applied here was cui-avatar__img--hidden.

Comment (bfbiggs) [reply src/lib/Avatar/index.js]: this is an insurance policy to make sure it can't possibly show up.

**Pull request merged**